lib/elixir/lib/calendar/datetime.ex

```diff
@@ -26,7 +26,7 @@ defmodule DateTime do
   You will notice this module only contains conversion
   functions as well as functions that work on UTC. This
   is because a proper DateTime implementation requires a
-  TimeZone database which currently is not provided as part
+  Time Zone database which currently is not provided as part
```

Comment:
I think this should probably just say "time zone". I'm not sure why it was capitalised like that.

Reply:
Agree. Changing it.